rust/agents/relayer/src/msg/pending_message.rs

```diff
@@ -142,6 +150,16 @@ impl PendingOperation for PendingMessage {
         self.ctx.destination_mailbox.domain()
     }
 
+    fn retrieve_status_from_db(&self) -> Option<PendingOperationStatus> {
+        match self.ctx.origin_db.retrieve_status_by_message_id(&self.id()) {
+            Ok(status) => status,
+            Err(e) => {
+                warn!(error=?e, "Failed to retrieve status for message");
```

Comment:
@tkporter FYI this will initially cause lots of warning logs, until the relayer retries every message once

Reply:
is it bc nothing will have been persisted yet?

I think `retrieve_status_by_message_id` gives Result<Option<_>> and it gives Ok(None) if there was no error reading the db but it just doesn't have an entry